README.md

```diff
@@ -10,10 +10,13 @@ SCL Ruby with SystemTap patches are required.
 
 ## Installation
 
-    yum -y install systemtap systemtap-runtime kernel-devel-`uname -r`
+    yum -y install gcc systemtap systemtap-runtime kernel-devel-`uname -r`
 
-## Usage
+    Create foreman-tracer script with: 
```

Comment:
Hey thanks, this is mis-formatted, can you make this a normal text? Then squash it into one commit please.

Did tracer help to you? I hope so. I am working on better monitoring of Katello :-)